src/components/ImageSearchResultTable.tsx

```diff
@@ -58,30 +99,45 @@ export const columns: ColumnDef<SearchableImageWithGameCard>[] = [
   {
     accessorKey: "tags",
     header: () => "Tags",
+    // cell: ({ row }) => {
```

Comment:
should we rm this comment?

Reply:
Yes.  Those comments should be removed.  Once I prove that the implementation works, I normally remove the original commented code, but had a few in flight.